st2common/st2common/models/db/stormbase.py

```diff
@@ -78,6 +78,23 @@ class StormBaseDB(StormFoundationDB):
     }
 
 
+class StormContentDB(StormBaseDB):
+    """Abstraction for a user content model from a pack."""
+    name = me.StringField(required=True)
+    pack = pack = me.StringField(
+        required=False,
+        help_text='Name of the content pack.',
+        unique_with='name')
+    file_uri = me.StringField(
```

Comment:
I read through some more code and see the benefit of `entry_point` https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/container/utils.py#L203. 

I guess we will have entry_point, artifact_uri and file_uri for SensorTypeDB. Other will gave file_uri.

I am open to renaming of file_uri to anything else more suitable.